src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -149,7 +154,9 @@ function getDrupalContent(buildOptions) {
     } catch (err) {
       buildOptions.drupalError = drupalData;
       log(err.stack);
-      log(JSON.stringify(drupalData));
+      if (drupalData) {
+        log(JSON.stringify(drupalData));
+      }
```

Comment:
Should we still have this? It seems like sometimes `drupalData` is so massive that it takes over the whole terminal

Reply:
Yeah, maybe we should. If there's an error in that pipeDrupalInfoMetalsmith function, there will be too much data logged to be useful.